titus-api/src/main/java/com/netflix/titus/api/jobmanager/JobParameterAttributes.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager;
+
+/**
+ * Constant keys for Job attributes.
+ */
+public final class JobParameterAttributes {
+    public static final String JOB_PARAMETER_ATTRIBUTES_ALLOW_CPU_BURSTING = "titusParameter.agent.allowCpuBursting";
```

Comment:
I'd vote for `parameter.titus.*` so we run less risk of someone matching the `titus*` prefix without the dot. That also allows for higher level systems to define their own set of parameters, e.g.: `parameter.spinnaker.*` etc.

Reply:
I am not opposed to this idea. I like `titus` being first so I can easily see that it is a titus concept but agree that it might be error prone since `titus.` is reserved.

Reply:
Are there going to titusParameter.server / titusParameter.controlPlane?

Reply:
There could be something used here to describe something to the control plane.